website/src/pages/documents/document.page.tsx

```diff
@@ -331,12 +337,20 @@ export const DocumentTitleHeader = (p: {
       {p.doc.title}
       {p.doc.date && ` (${p.doc.date.year})`}{" "}
     </h1>
+        
+    
+
     <div className={css.bottomPadded}>
       {p.showDetails ? (
         <Link href={documentDetailsRoute(p.doc.slug!)}>View Details</Link>
       ) : (
         <Link href={documentRoute(p.doc.slug!)}>View Contents</Link>
       )}
+      {!p.doc.audioRecording && (
+        <div id="no-audio-message">
+        {!isMobile ? <strong>No Audio Available</strong> : null}
+        </div>
+      )}
```

Comment:
Hmm it looks like this can be simplified by putting the `!isMobile` clause in the first condition, right?